.github/workflows/mkdocs-master.yml

```diff
@@ -0,0 +1,38 @@
+name: mkdocs
```

Comment:
You might need to give them unique names here, so we can distinguish them in the Github UI where the runs are shown.
```suggestion
name: mkdocs-master
```

Reply:
I set unique names on all PRs